src/community/geofence-server/src/test/java/org/geoserver/geofence/rest/RulesRestControllerTest.java

```diff
@@ -57,50 +63,7 @@ public void testInsertUpdateDelete() {
         rule.setLayer("layer");
         rule.setAccess("ALLOW");
 
-        JaxbRuleList rules =
```

Comment:
What is going on here?

Reply:
Yes, I have places the cleanup code into the utility method of the same class

protected long prepareGeoFenceTestRules(JaxbRule rule)

That one checking and cleaning up all the previously created rules.